src/components/Navigation.js

```diff
@@ -1,18 +1,25 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 
 const Navigation = () => {
   return (
     <div>
       <div className="App">
         <h1>React Router Mini</h1>
         <div>
-          <a href="">Home</a>
+          <Link to="/">
+            <a href="">Home</a>
+          </Link>
         </div>
         <div>
-          <a href="">About</a>
+          <Link to="/about">
+            <a href="">About</a>
```

Comment:
you don't need to mix the Link with the a tag. In fact, I'd advise against it. This should look like this:

<Link to="/about">About</Link>